package.json

```diff
@@ -68,6 +68,7 @@
     "reactjs-mixin": "0.0.2",
     "redux": "3.3.1",
     "reflect-metadata": "0.1.10",
+    "rxjs": "5.4.3",
```

Comment:
Since we are adding a dependency here shouldn't we have additions to the npm-shrinkwrap? currently I only see removals of `fsevents` which is unfortunate that it landed on master again but okay. 

Just want to make sure this is right.

Reply:
I asked that here in person, rxjs was already a dependency of one of our dependencies 👍

Reply:
It's there already: https://github.com/dcos/dcos-ui/blob/ac7bb50cab9ed517f89c2abe3558fb9ab4f6961c/npm-shrinkwrap.json#L8555

Reply:
good!